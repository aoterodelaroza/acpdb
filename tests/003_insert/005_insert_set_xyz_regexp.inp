## check: 005_insert_set_xyz_regexp.out -a1e-10
## delete: 005_insert_set_xyz_regexp.db
## labels: regression quick

verbose
system rm -f 005_insert_set_xyz_regexp.db
connect 005_insert_set_xyz_regexp.db

insert litref bibtex ../zz_source/bsip2.bib
insert set s22x5
 description A description.
 litrefs bsip1 boys1970
 xyz ../zz_source/s22_xyz/ .*\.xyz
end
