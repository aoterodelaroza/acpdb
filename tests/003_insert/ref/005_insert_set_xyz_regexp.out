* SYSTEM: rm -f 005_insert_set_xyz_regexp.db

* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 005_insert_set_xyz_regexp.db
Creating skeleton database 

* INSERT: insert data into the database (LITREF)

* INSERT: insert data into the database (SET)

